integration-tests/tests/update_test.go

```diff
@@ -73,7 +73,8 @@ func (s *updateSuite) TestUpdateToSameReleaseAndChannel(c *check.C) {
 		common.Reboot(c)
 	} else if common.AfterReboot(c) {
 		common.RemoveRebootMark(c)
-		c.Assert(common.GetCurrentUbuntuCoreVersion(c) > common.GetSavedVersion(c),
-			check.Equals, true)
+		currentVersion := common.GetCurrentUbuntuCoreVersion(c)
```

Comment:
We are no longer putting anything new in common, and we are slowly splitting it as part of this card: https://trello.com/c/F2CLsFwK/227-split-the-common-package-by-functionality

most of the contents should be moved to base_test.go.